pkg/cover/server.go

```diff
@@ -33,46 +33,46 @@ import (
 	"k8s.io/test-infra/gopherage/pkg/cov"
 )
 
-// DefaultStore implements the IPersistence interface
-var DefaultStore Store
+//// DefaultStore implements the IPersistence interface
+//var DefaultStore Store
 
 // LogFile a file to save log.
 const LogFile = "goc.log"
 
-func init() {
-	DefaultStore = NewFileStore()
+type ServiceGoc struct {
```

Comment:
no need with suffix `Goc` which seems redundancy。suggest to just use `Service` name.